xbmc/music/tags/MusicInfoTag.h

```diff
@@ -159,6 +160,7 @@ class CMusicInfoTag final : public IArchivable, public ISerializable, public ISo
   void SetStationName(const std::string& strStationName); // name of online radio station
   void SetStationArt(const std::string& strStationArt);
   void SetSongVideoURL(const std::string& songVideoURL); // link to video of song
+  void SetChapterMarks(const std::map<int, std::vector<std::string>>& chapters);
```

Comment:
Would it make sense to introduce a speaking type using `using`, so it is clear to the reader what the map holds? Maybe something like `ChaptersMap`?

Reply:
See previous reply.